plugins/gatsby-remark-dvc-linker/simpleLinker.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-env node */
+
+const { createLinkNode } = require('./helpers')
+
+const entries = require('../../content/linked-terms')
+
+const useMatcher = (matcher, item) => {
+  switch (typeof matcher) {
+    case 'string':
+      return item === matcher
+    case 'object':
+      if (Array.isArray(matcher))
+        return matcher.find(submatcher => useMatcher(submatcher, item))
+      if (matcher instanceof RegExp) return matcher.match(item)
+    default:
+      throw `gatsby-remark-dvc-linker simpleLinker given bad matcher of type "${typeof matcher}"`
```

Comment:
looks like linters are not applied to this file?

Reply:
Maybe? I'm not sure what format rules are being broken, as running prettier manually doesn't force make any changes.

Reply:
ah, I see it the same as https://github.com/iterative/dvc.org/issues/1112 probably